### ISSUE TITLE
bedtools intersect -wao with -b file empty gives inconsistent output

### ISSUE THREAD

**Issue by wtwhite** [arq5x/bedtools2#724]
`bedtools intersect -wao` behaves inconsistently: Normally, when none of the intervals in the `-b` intersect any intervals in the `-a` file, it outputs each line from the `-a` file followed by a `.`, two `-1`s and a `0` in additional tab-separated columns; but when the `-b` file is empty, it instead appends two tabs and a `0` to each of these lines.  This necessitates special-case detection and handling of empty `-b` files in pipelines.

The following snippet reproduces the problem in v2.27.1 (I quickly scanned the commits for v2.28.0 and didn't see anything obviously relevant).  The two `bedtools` invocations should produce the same output, since in both cases there were no intervals in common between the two input files, but they don't:

```bash
$ echo -e 'chr1\t100\t150' > truth.bed
$ echo -e 'chr2\t500\t800' > complete-miss.bed
$ touch empty.bed
$ bedtools intersect -a truth.bed -b complete-miss.bed -wao
chr1	100	150	.	-1	-1	0
$ bedtools intersect -a truth.bed -b empty.bed -wao
chr1	100	150		0
```

Possibly related to #546, though there are a few differences (no crash, `intersect` instead of `merge`, problem occurs when the `-b` file is empty, not the `-a` file).

**Comment by jmarshall** [comment]
This was fixed by bdfc47c53ba7ed84bbacd04c33e58c6084432073, which is in 2.28.0.

**Comment by wtwhite** [comment]
I just downloaded this and am happy to confirm that it now works properly on my test above.

Thanks @jmarshall for the quick response!
